docs/binary.md

```diff
@@ -52,12 +47,8 @@ It is just like counting in decimal except we reach 10 much sooner.
 | ???          | But then there is no symbol for 2 ... what to do?  |
 
 
-<<<<<<< Updated upstream
-<h2>Well how do we count in decimal?</h2>
-=======
 ## Well how do we count in Decimal?
```

Comment:
Another small attention to detail, it is in best practices to have a smaller dimension heading under a given heading instead of having the heading and sub-heading as the same dimension. For example, < h2 > level headings should not have sub-headings of < h2 > level, instead they should be a dimension (or more) smaller, like < h3 > or < h4 >

Over here, _How do we count using binary?_  is a subheading to _Binary Counting_ and thus should not be at par with it's parent-heading level. I would suggest keep the subheadings at < h3 > i.e. `###` level and the headings at < h2 > i.e. `##` level